src/test/java/uk/gov/hmcts/reform/ccd/ApplicationExecutorTest.java

```diff
@@ -36,9 +41,17 @@ class ApplicationExecutorTest {
     @Mock
     private CaseDeletionService caseDeletionService;
 
+    @Mock
+    private ParameterResolver parameterResolver;
+
     @InjectMocks
     private ApplicationExecutor applicationExecutor;
 
+    @BeforeEach
```

Comment:
You don't need this parameterResolver setup here because it's already injected by mockito.

Reply:
Removed the setup